docs/demonstration/schoology-e2e.ipynb

```diff
@@ -1613,7 +1613,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.9.4 64-bit ('edfi')",
```

Comment:
QUERY: Are the updates to `display_name` and `vscode` intended?

Reply:
These updates were made by the tool, and I don't see any harm in them. You can double-check that it works fine still from `poetry run jupyter notebook` - I'm quite confident that it won't be a problem.